### ISSUE TITLE
Missing the opportunity to select between merge/upload/download in smartphone app.

### ISSUE THREAD

**Issue by SchwarzerA** [floccusaddon/floccus#1400]
### Describe the feature you'd like to request

Floccus, deployed in a browser, offers three types of synchronization. In my opinion it would also be a valuable feature in the smartphone app. More than 99 % of my usage is a simple download of bookmarks maintained elsewhere.

### Describe the solution you'd like

An opportunity to select one of a) download (default), b) upload or c) merge your bookmarks.

### Describe alternatives you've considered

none

**Comment by Uatschitchun** [comment]
This is already possible to select in the app!?

**Comment by marcelklehr** [comment]
Yes, while the app does not offer an on-demand choice of buttons for one-off pull/push sync, you can set which method you want in the settings.
